src/libraries/Microsoft.PowerFx.Connectors/ConfigExtensions.cs

```diff
@@ -27,7 +26,8 @@ public static class ConfigExtensions
         /// <param name="httpClient">Required iff we want to invoke the API. A client to invoke the endpoints described in the api document. This must handle auth or any other tranforms the API expects.</param>
         /// <param name="cache">A cache to avoid redundant HTTP gets.</param>
         /// <param name="numberIsFloat">Number is float.</param>
-        public static IReadOnlyList<FunctionInfo> AddService(this PowerFxConfig config, string functionNamespace, OpenApiDocument openApiDocument, HttpMessageInvoker httpClient = null, ICachingHttpClient cache = null, bool numberIsFloat = false)
+        /// <param name="maxRows">Max rows.</param>
+        public static IReadOnlyList<FunctionInfo> AddService(this PowerFxConfig config, string functionNamespace, OpenApiDocument openApiDocument, HttpMessageInvoker httpClient = null, ICachingHttpClient cache = null, bool numberIsFloat = false, int maxRows = 1000)
```

Comment:
>1000 [](http://example.com/codeflow?start=260&length=4)

Can we set this as a constant somewhere? There are other places that this value will be referenced from.

Reply:
Fixed